app/components/show/table_heading_component.rb

```diff
@@ -2,11 +2,21 @@
 
 module Show
   # Component for rendering a table heading.
-  class TableHeadingComponent < Elements::HeadingComponent
-    def initialize(**args)
-      args[:level] = :h2
```

Comment:
What happened to the `h2` concern here? No longer relevant due to other changes?

Reply:
Yes. Before TableHeadingComponent was a subclass of Elements::HeadingComponent. It now renders Elements::HeadingComponent and provides the level as an argument.